tests/standard/1.in.json

```diff
@@ -0,0 +1,37 @@
+{
+  "BoardState": {
```

Comment:
This is a direct marshal/unmarshal format, and these structs don't have specific `json` 'identifiers` defined. So it definitely ties the format of this JSON to the current structure of the underlying structs

Is this ok?
Or would you rather have a more 'standard' looking JSON with lowercase names? If so would it be ok to add the `json` identifiers to the struct like I saw in the CLI example? Or is there a better way to do this?